zajecia/src/stringIBigDecimal/MainBigDecimal.java

```diff
@@ -0,0 +1,224 @@
+package stringIBigDecimal;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.Collections;
+
+public class MainBigDecimal {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+//        /*
+//        Zadanie 11
+//Formatowanie ceny z użyciem BigDecimal
```

Comment:
podziel to na osobne klasy/metody, strasznie ciezko sie czyta takie pokomentowane wszystko, jako komenatrz powinny zostac tylko tresci zadan, odpowiedzi na pytania i jakies wnioski z zadan, kod do zadan powinien byc w odkomentowanej formie